README.md

```diff
@@ -1,2 +1,26 @@
-# gpii-devpmt
-Development Preferences Test Tool and other Management Tools
+# Development Preferences Test Tool and other Management Tools
+
+This project is the foundation for the DPTT and other comprehensive preferences
```

Comment:
Please add links to wireframes and any other specification material explaining what on earth this is

Reply:
these were added the readme a while back